docs/examples/eandt.py

```diff
@@ -0,0 +1,70 @@
+from gladier import GladierBaseClient, generate_flow_definition, GladierBaseTool
+from pprint import pprint
+
+
+def makeFiles(**data):
+    import os
+    input_path = data['make_input']
+
+    if '~' in input_path:
+        input_path = os.path.expanduser(input_path)
+
+    with open(input_path+"/file1.txt", "w") as file1:
+        file1.write("This is file no. 1")
+
+    with open(input_path+"/file2.txt", "w") as file2:
+        file2.write("This is file no. 2")
+
+    with open(input_path+"/file3.txt", "w") as file3:
+        file3.write("This is file no. 3")
```

Comment:
Joining paths here should use `os.path.join` or `pathlib.Path`. It could also be golfed a bit:

```
    input_path = pathlib.Path(data['make_input'])
    for number in range(3):
        with open(input_path / f'file{number}.txt', 'w') as f:
            f.write(f'This is file no. {number}')

    return input_path
```